jsoc/gsoc/javis.md

```diff
@@ -1,69 +1,44 @@
-The following list are the smaller tasks one could work on:
-
-- One of the bigger missing features is the lack of combining several objects into a layer. [Issue #75](https://github.com/Wikunia/Javis.jl/issues/75)
-
-- To improve the user experience it will be helpful to ease object positioning based on other objects. [Issue #130](https://github.com/Wikunia/Javis.jl/issues/130)
-
-- For visual appeal, morphing shapes into one another shall be improved as it's currently an undocumented and unfinished feature. [Issue #286](https://github.com/Wikunia/Javis.jl/issues/286)
-
-- To bring Javis and Julia closer to the broader audience we are interested in the ability of live streaming animations to platforms like Twitch. [Issue #91](https://github.com/Wikunia/Javis.jl/issues/91)
+- **[Giovanni Puccetti](https://gpucce.github.io)** - [Zulip](https://julialang.zulipchat.com/) (username: Giovanni)
 
-## Graph and networks
+- **[Arsh Sharma](https://sov-trotter.github.io/blog/)** - [Zulip](https://julialang.zulipchat.com/) (username: Arsh Sharma)
 
-**Mentors**: Ole Kröger, Jacob Zelko
-
-**Recommended skills**: Knowledge about graph theory and LightGraphs.jl
-
-**Difficulty:** Hard
-
-**Description**: Javis could be a powerful platform to easily animate problems and their solutions in a variety of different fields.
-Currently, Javis lacks the ability to visualize graphs.
-The goal for this project would be to add graph support to Javis by supporting interoperability with [LightGraphs.jl](https://github.com/JuliaGraphs/LightGraphs.jl).
-The animation of flows and shortest path is something that's extremely valuable for teaching as well as in practical analysis of graph networks.
-To learn more about the current thoughts surrounding this problem, check this [issue](https://github.com/Wikunia/Javis.jl/issues/41) for more information. 
-
-## Linear algebra
-
-**Mentors**: Ole Kröger, Jacob Zelko
-
-**Recommended skills**: Basic to intermediate knowledge about linear algebra.
-
-**Difficulty:** Easy 
-
-**Description**: Linear algebra is of invaluable importance all across different fields of mathematics and engineering.
-Enabling the easy creation of visualizations regarding rotations, matrices and other concepts is helpful in educating students about this amazing branch mathematics.
-Here are a few issues related to tasks that could be worked on to bring about this capability:
-
-- Vectors are foundational to linear algebra, help Javis visualize them! [Issue #31](https://github.com/Wikunia/Javis.jl/issues/31)
+Thanks for your interest! :tada:
 
-- Drawing backgrounds such as grids can assist in easy viewing of complicated mathematical operations such as rotations. [Issue #38](https://github.com/Wikunia/Javis.jl/issues/38)
+## Possible Project Ideas
+
+- Explore possible integrations with Makie?
+	- Creating 3D animations in Javis
+	- Support plotting integration if possible
+- Graph network interaces for Javis
```

Comment:
Maybe @gpucce can you comment a bit on the current state of the graph network capabilities?
I think currently `LinearAlgebra` isn't too well defined of what we want. We might want to discuss this but I would prefer to fix some other issues first and either start with some ideas ourselves or define those in more detail as too broad topics are probably getting out of hand in GSoC.

Reply:
For the graph package I think I can do it myself in not too long. Although perhaps I would need to talk about it with you guys to understand what level of functionality would be minimal required, however in the next period I will be super busy.

On the other end I think for someone else building upon what I did so far would be more or less like starting from scratch since there is still not much documentation and the graph functionality part is more easily learned reading through Graphs.jl or even better GraphMakie.jl and NetworksLayout.jl.

I am happy to help with that during summer or do it myself I don't have any issue with either solution.